.github/workflows/TestOnPR.yml

```
@@ -1,4 +1,4 @@
-name: "CI - On main/dev/release branches"
+name: "PR checks"
```

Should this be PR Checks?
```suggestion
name: "PR Checks"
```